### ISSUE TITLE
Error running grafana exporter sample 

### ISSUE THREAD

**Issue by manueligno78** [jakubplichta/grafana-dashboard-builder#138]
Hi, I have installed grafana-dashboard-builder, (before I have installed xcode, xcode command line tools and python)
When I run:
```
grafana-dashboard-builder -p ./samples/project.yaml --exporter grafana --config ./samples/config.yaml

2019-03-01 17:20:56,989 INFO Loading component type <class 'grafana_dashboards.components.projects.Project'>

2019-03-01 17:20:56,989 INFO Loading component type <class 'grafana_dashboards.components.annotations.AnnotationsItemBase'>

2019-03-01 17:20:56,989 INFO Loading component type <class 'grafana_dashboards.components.axes.YaxesItemBase'>

2019-03-01 17:20:56,989 INFO Loading component type <class 'grafana_dashboards.components.links.LinksItemBase'>


2019-03-01 17:20:56,989 INFO Loading component type <class 'grafana_dashboards.components.targets.TargetsItemBase'>

2019-03-01 17:20:56,989 INFO Loading component type <class 'grafana_dashboards.components.panels.PanelsItemBase'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.rows.RowsItemBase'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.templates.TemplatesItemBase'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.dashboards.Dashboard'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.annotations.Annotations'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.axes.Yaxes'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.links.Links'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.targets.Targets'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.panels.Panels'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.rows.Rows'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.templates.Templates'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.annotations.Annotation'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.axes.Yaxis'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.links.DashboardLink'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.links.AbsoluteLink'>

2019-03-01 17:20:56,990 INFO Loading component type <class 'grafana_dashboards.components.targets.GraphiteTarget'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.targets.PrometheusTarget'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.targets.InfluxdbTarget'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.panels.Graph'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.panels.SingleStat'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.panels.Table'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.panels.Text'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.panels.Dashlist'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.rows.Row'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.templates.Query'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.templates.EnumeratedTemplateBase'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.templates.DatasourceTemplate'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.templates.CustomTemplate'>

2019-03-01 17:20:56,991 INFO Loading component type <class 'grafana_dashboards.components.templates.IntervalTemplate'>

2019-03-01 17:20:57,031 INFO Component 'dict_keys(['default-dashboard'])' does not have 'name' attribute, skipping

2019-03-01 17:20:57,032 INFO Processing project 'Example project'

2019-03-01 17:20:57,033 INFO Uploading dashboard 'overview' to http://localhost:3000
Traceback (most recent call last):
  File "/usr/local/bin/grafana-dashboard-builder", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/grafana_dashboards/cli.py", line 102, in main
    project_processor.process_projects(projects, context)
  File "/usr/local/lib/python3.7/site-packages/grafana_dashboards/exporter.py", line 57, in process_projects
    processor.process_dashboard(project.name, dashboard_name, json_obj)
  File "/usr/local/lib/python3.7/site-packages/grafana_dashboards/client/grafana.py", line 46, in process_dashboard
    self._connection.make_request('/api/dashboards/db', body)
  File "/usr/local/lib/python3.7/site-packages/grafana_dashboards/client/connection.py", line 67, in make_request
    response_body = self._opener.open(request).read()
  File "/usr/local/Cellar/python/3.7.2_2/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 523, in open
    req = meth(req)
  File "/usr/local/Cellar/python/3.7.2_2/Frameworks/Python.framework/Versions/3.7/lib/python3.7/urllib/request.py", line 1247, in do_request_
    raise TypeError(msg)

TypeError: POST data should be bytes, an iterable of bytes, or a file object. It cannot be of type str.
```

**Comment by jakubplichta** [comment]
@manueligno78 thanks for the report. It looks like added support for py3 was not as smooth as I'd expected and I haven't tested some parts.

**Comment by jakubplichta** [comment]
@manueligno78 patched version released as v0.5.0a2

**Comment by manueligno78** [comment]
It works now! Thank you
